code/game/machinery/adv_med.dm

```diff
@@ -420,8 +420,7 @@
 
 		dat += "<hr>"
 
-		var/blood_percent =  round((occupant.blood_volume / BLOOD_VOLUME_NORMAL))
-		blood_percent *= 100
+		var/blood_percent =  round(occupant.blood_volume /BLOOD_VOLUME_NORMAL * 100, 1)
```

Comment:
```suggestion
		var/blood_percent =  round(occupant.blood_volume / BLOOD_VOLUME_NORMAL * 100, 1)
```
also please use parentheses here to really make clear what the numerator and denominator of this is

Reply:
Will do!